explorer/explorertypes.go

```diff
@@ -35,6 +35,7 @@ var blockchainSyncStatus = new(syncStatus)
 
 // BlockBasic models data for the explorer's explorer page
 type BlockBasic struct {
+	WindowIndx     int64  `json:"windowIndex"`
```

Comment:
Please move this down below next to `FreshStake`, but mainly just not at the top.  Also, same comment about window ambiguity.